keras_nlp/src/models/gemma/gemma_presets.py

```diff
@@ -215,4 +215,34 @@
         },
         "kaggle_handle": "kaggle://keras/gemma2/keras/gemma2_instruct_27b_en/1",
     },
+    "shieldgemma_2b_en": {
+        "metadata": {
+            "description": "2 billion parameter, 26-layer, ShieldGemma model.",
+            "params": 2614341888,
+            "official_name": "Gemma",
+            "path": "gemma",
+            "model_card": "https://www.kaggle.com/models/google/gemma",
+        },
+        "kaggle_handle": "kaggle://google/shieldgemma/keras/shieldgemma_2b_en/1",
+    },
+    "shieldgemma_9b_en": {
+        "metadata": {
+            "description": "9 billion parameter, 42-layer, ShieldGemma model.",
+            "params": 9241705984,
+            "official_name": "Gemma",
+            "path": "gemma",
+            "model_card": "https://www.kaggle.com/models/google/gemma",
+        },
+        "kaggle_handle": "kaggle://google/shieldgemma/keras/shieldgemma_9b_en/2",
```

Comment:
Only 9b is version 2?

Reply:
oops, no i forgot we had to patch the regular gemma 9b and copy pasted. thanks for catching!!